libs/deps/ost-wrapper.js

```diff
@@ -0,0 +1,11 @@
+import { loadScript, getConfig } from '../utils/utils.js';
+
+const { miloLibs, codeRoot, env } = getConfig();
+const base = miloLibs || codeRoot;
+
+export const initOst = async (el) => {
+  if (!window.ost) {
+    await loadScript(`${base}/deps/offer-selector-tool.js`);
+  }
+  window.ost.openOfferSelectorTool();
```

Comment:
I remember @auniverseaway suggested to initialize ost from a wrapper file in deps folder since technically ost is considered a milo block. If needed I can move this some place else.